inc/saiswitch.h

```diff
@@ -3692,6 +3692,11 @@ typedef enum _sai_switch_stat_t
  */
 #define SAI_KEY_INIT_CONFIG_FILE                  "SAI_INIT_CONFIG_FILE"
 
+/**
+ * @def SAI_KEY_NHG_HIERARCHICAL_NEXTHOP
+ */
```

Comment:
For this one, please add a comment on usage, similar to the SAI_KEY_BOOT_TYPE. For example if 0 means hierarchical nhg are disabled and 1 means hierarchical nhg are enabled.

Can take a follow-up item to check on means to read it back.